docs/self-managed/platform-deployment/helm-kubernetes/update.md

```diff
@@ -82,14 +82,14 @@ helm upgrade <RELEASE_NAME> charts/camunda-platform/ \
 ```
 
 :::note
-If you have specified on the first installation certain values, you have to specify them again on the upgrade either via `--set` or the values file and the `-f` flag.
+If you have specified on the first installation certain values, you have to specify them again on the update either via `--set` or the values file and the `-f` flag.
 :::
 
-For more details on the Keycloak upgrade path, you can also read the [Bitnami Keycloak upgrade guide](https://docs.bitnami.com/kubernetes/apps/keycloak/administration/upgrade/).
+For more details on the Keycloak update path, you can also read the [Bitnami Keycloak upgrade guide](https://docs.bitnami.com/kubernetes/apps/keycloak/administration/upgrade/).
```

Comment:
Since we don't control Keycloak, let's stick to the terminology they use.

Reply:
Agreed here. Adjusting in latest commit 👍